app/models/post.rb

```diff
@@ -0,0 +1,14 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :likes, foreign_key: :post_id, dependent: :delete_all
+  has_many :comments, foreign_key: :post_id, dependent: :delete_all
+
```

Comment:
- [OPTIONAL] I don't think it's necessary to add the `foreign_key` on the has_many: comments and likes models relationship. 🤔